src/main/java/dk/kb/present/util/DsPresentClient.java

```diff
@@ -175,13 +188,24 @@ public ContinuationInputStream<Long> getRecordsJSON(String origin, Long mTime, L
      */
     public ContinuationInputStream<Long> getRecordsRawJSON(
             String origin, Long mTime, Long maxRecords, Boolean asJsonLines) throws IOException {
-        URI uri = UriBuilder.fromUri(serviceURI)
-                .path("recordsraw")
-                .queryParam("origin", origin)
-                .queryParam("mTime", mTime == null ? 0L : mTime)
-                .queryParam("maxRecords", maxRecords == null ? 10 : maxRecords)
-                .queryParam("asJsonLines", asJsonLines != null && asJsonLines)
-                .build();
+        URI uri;
+        try {
+            uri = new URIBuilder(serviceURI)
+                    .setPath("records")
```

Comment:
Should be recordsraw right?

Reply:
Kinda important to call the right endpoint. Thanks!